integration-cli/docker_cli_wait_test.go

```diff
@@ -120,3 +121,45 @@ func TestWaitBlockedExitRandom(t *testing.T) {
 
 	logDone("wait - blocking wait with random exit code")
 }
+
+func TestWaitKilledContainerNoWait(t *testing.T) {
+	defer deleteAllContainers()
+
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "/bin/cat")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		t.Fatal(out, err)
+	}
+	containerID := strings.TrimSpace(out)
+
+	killCmd := exec.Command(dockerBinary, "kill", containerID)
+	out, _, err = runCommandWithOutput(killCmd)
+	if err != nil {
+		t.Fatal(out, err)
+	}
+
+	errchan := make(chan error)
+	var wg sync.WaitGroup
+	defer wg.Wait()
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		waitCmd := exec.Command(dockerBinary, "wait", containerID)
+		out, _, err := runCommandWithOutput(waitCmd)
+		if err != nil {
```

Comment:
seems like errchan <- err is enough without check and then you'll get nil in your e